components/tyk-gateway/values.yaml

```diff
@@ -233,3 +239,7 @@ gateway:
   #    filename: test.txt
   #    mountPath: /opt/test
   mounts: []
+
+  # used to decide whether to send the results back directly to Tyk without a hybrid pump
+  # if you want to use a hybrid pump, please change "rpc" to ""
+  analyticsConfigType: "rpc"
```

Comment:
I think default for gateway component should be ""
(Set empty for a Self-Managed installation or rpc for multi-cloud.)

Reply:
Fixed